container/apptabs.go

```diff
@@ -341,9 +341,8 @@ func (r *appTabsRenderer) buildTabButtons(count int) *fyne.Container {
 		item := r.appTabs.Items[i]
 		button, ok := r.buttonCache[item]
 		if !ok {
-			index := i // capture
 			button = &tabButton{
-				onTapped: func() { r.appTabs.SelectIndex(index) },
+				onTapped: func() { r.appTabs.Select(item) },
```

Comment:
Hmm. I thought a bit more about this. The fix works but the change also results in that we loop through all tab items to see if the tab exists when we tap a it. That isn’t very elegant or performant.